README.md

```diff
@@ -102,12 +102,12 @@ the main network, but with play-Ether only.
 $ geth --goerli console
 ```
 
-The `console` subcommand has the exact same meaning as above and they are equally
-useful on the testnet too. Please, see above for their explanations if you've skipped here.
+The `console` subcommand has the same meaning as above and is equally
+useful on the testnet too. Please, see above for their explanations if you've skipped them here.
```

Comment:
```suggestion
useful on the testnet too. Please, see above for their explanations if you've skipped them here.
```
I think intent was "if you have skipped over the sections above, down to here"

Reply:
Not sure, deleted that.